addons/core/package.json

```diff
@@ -42,7 +42,8 @@
     "ember-loading": "^2.0.0",
     "ember-truth-helpers": "^3.0.0",
     "filesize": "^10.0.7",
-    "intl-unofficial-duration-unit-format": "^3.1.0"
+    "intl-unofficial-duration-unit-format": "^3.1.0",
+    "lodash": "^4.17.21"
```

Comment:
Should lodash have been removed from admin and desktop now?

Reply:
Yes. I will do that now. Thanks for the reminder.